nym-vpn-core/nym-vpn-lib/src/mobile/ios/dns64.rs

```diff
@@ -54,7 +54,7 @@ impl Dns64Resolution for WgPeer {
 }
 
 /// Re-resolve an endpoint with dns64
-fn reresolve_endpoint(endpoint: SocketAddr) -> Result<SocketAddr> {
+pub(crate) fn reresolve_endpoint(endpoint: SocketAddr) -> Result<SocketAddr> {
```

Comment:
Can we please keep this untouched. This is not meant to be used outside this file.

Reply:
Yeah! I'll fix this!